Makefile

```diff
@@ -23,26 +27,18 @@ modules_install:
 	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) modules_install
 
 #Performs a dkms install
-install: dkms-prepare
-	dkms install -m mtcadummy -v ${MTCADUMMY_PACKAGE_VERSION} -k $(KVERSION)
+kernel_install:
+	$(MAKE) -C $(KERNEL_SRC) V=1 M=$(PWD) install
```

Comment:
This does not perform a DKMS install, does it? It installs into the current kernel, but if I get a kernel upgrade I am left without driver.

Reply:
In addition, there is no uninstall option.

Reply:
The idea is that this converts the whole source into a plain kernel driver, and the DKMS is done in DebianBuildScripts.

Reply:
I get the idea. But then you only get dkms with the debian package. Before you got dkms from source.